app/controllers/api/providers_controller.rb

```diff
@@ -85,6 +85,21 @@ def import_vm_resource(type, id = nil, data = {})
       end
     end
 
+    def options
+      if params[:ems_type].blank?
```

Comment:
See also https://github.com/ManageIQ/manageiq-api/pull/3#discussion_r132243534 for my explanation there - I think I'd prefer not including this unless there is a distinct need for it. If that is the case, I suspect OPTIONS might not be the right solution